### DISCUSSION TITLE
autodoc // auto-deploy

### DISCUSSION THREAD

**Discussion by TfedUD** [PH-Tools/honeybee_ph#10] [General]
it looks like everything is basically good to go for autodoc,
    We just need somewhere for it to "go" i.e a ph-tools gitpages or somthing. 
   due to a "flat dir with multiple packages" I just gotta specify the package locations; and tell it where to 'deploy the docs to' otherwise idk if it worked lol
but yeah other than that; should be good to go

**Comment by ed-p-may** [comment]
sweet!!!!! What about [R](https://readthedocs.org/) ? I think they do free hosting for docs, right? I asked Chris and Mostapha if they wanted all the docs for plugins  hosted through their site, and they said for now we should just do it indepedanlty. We can certianly host through the passivehousetools.com site as well? I can send you the login info for that if you want to try there?

**Reply by TfedUD** [reply]
cool! I'll check it out!
